### ISSUE TITLE
Migrate miniverse to AWS

### ISSUE THREAD

**Issue by landreev** [IQSS/dataverse#4477]
For various reasons we decided to handle this migration separately from moving the "workhorse" services and the data (completed on Feb. 26 2018). 

The miniverse move is much simpler; because the application is fairly compact and there is no underlying data of its own. 

The remaining tasks include setting up django environment on the AWS node (need to verify where it is with the LTS), and deploy the miniverse application, following the instructions in Raman's documentation; 

Work with the LTS to verify all the networking, ports, etc. are properly open and the app can access the production database; 

Test everything; 

Request the DNS change for services.dataverse.harvard.edu, currently pointing to dvn-60ox-app-1.lib.harvard.edu, to the cloudshied address for the AWS server (dvn-h-prod-pub.gslb.harvard.edu)

This is a day of (my) work.

**Comment by landreev** [comment]
The server is dvn-cloud-miniverse.lib.harvard.edu; it's an 8G AWS instance.
I did some heavier lifting on this last week - installed the right version of Django and figured out the httpd configuration changes that were needed to reflect the different version of Apache (2.4 vs 2.2), made sure the OS user "miniverse" is properly set up. 
Still need to deploy and test the actual miniverse app and verify that it's producing legit metrics, before we can ask LTS to switch the DNS record "services.dataverse.harvard.edu" to this new host. But it appears to be very doable, with the estimate "3" being about right. 
I'm pretty confident we should go ahead and migrate it, and then decide if we want to retire it eventually. (Would be nice to have replacement technology already in place, before making the retirement decisions).

**Comment by landreev** [comment]
Let's get this thing out of the way. I'm putting it into QA directly (since there's nothing to review). 
I'm assuming QA for this is to compare it to the old prod. miniverse running on services.dataverse.harvard.edu and see if anything is broken.

The new, AWS location:
https://dvn-cloud-miniverse.lib.harvard.edu/
Metrics: 
https://dvn-cloud-miniverse.lib.harvard.edu/miniverse/metrics/basic-viz/last12
(you'll have to accept a cert exception; since the name "dvn-cloud-miniverse" is not in the cert; but once the DNS record is switched and it becomes services.dataverse.harvard.edu, the problem will go away).

**Comment by kcondon** [comment]
Leonid, I can only reach the new miniverse from on campus or via vpn, right?

**Comment by landreev** [comment]
Yes, it's the same policy as the other cloud nodes, only accessible from our local subnets. 

The cloudshield/ELB addresses, that should be accessible from everywhere, are already set up. But, according to the latest email from Paul, they are not working yet. (It wasn't clear why exactly - seems like an ongoing investigation; see that thread for info)

**Comment by landreev** [comment]
OK, the ELBs should be working now: 

http://dvn-h-prod-pub.gslb.harvard.edu   - should be visible from anywhere, but requires the certificate exception, since the .gslb name cannot be added to the cert; 

https://dvn-h-prod.hz.lib.harvard.edu/ - proper cert, but only visible from Harvard. 
(same deal as with the app nodes).
